### DISCUSSION TITLE
Declare that application supports the Share contract

### DISCUSSION THREAD

**Discussion by EvgenPervenenko** [wixtoolset/issues#7245] [Questions]
Hi, community, in windows 8 added a feature share content, is it possible to declare that application is supporting share content? Is it possible to add application to share dialog applications list?

Description:
https://blogs.windows.com/windowsdeveloper/2021/07/19/extending-the-context-menu-and-share-dialog-in-windows-11/#:~:text=The%20share%20dialog%20has%20also%20been%20improved%20in%20Windows%2011.%C2%A0
 
Win documentation:
https://learn.microsoft.com/en-us/previous-versions/windows/apps/hh871367(v=win.10)

**Comment by robmen** [comment]
Yes. From the blog post, you want to follow this bullet point:

> Apps extend the menu with IExplorerCommand + app identity. Unpackaged Win32 apps can use [Sparse Manifests](https://github.com/microsoft/AppModelSamples/tree/master/Samples/SparsePackages). IExplorerCommand support extends back to Windows 7.
